### PR TITLE
Align mqtt document with the current project set-up

### PR DESCRIPTION
**Pull request by relu91**
After #109 has landed, few things have changed in the project structure. In particular, every new protocol binding template docume should follow a defined structure and inside the same folder few other file should be provided:
- context.jsonld: needed for roundtripping TDs that are using the binding template
- mqtt.schema.json: needed for validation
- mapping.ttl: shacl file also needed for validation/ rendering spec document
- Other optional (but I would say reccomanded) files:
  * template.sparq and index.template.html: used if the document can be generated from the ontology and its shacl file. 
  
This PR aligns mqtt document to the new structure.

### CODE REVIEW

**Comment by egekorkan** [comment]
This looks good. I think you can think of adding yourself as an editor if you are planning to continue the work.

Note for others: This is mainly a restructuring not actual change of the content.

File: bindings/protocols/mqtt/index.template.html [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+            <td><code>mqv:optionValue</code></td>
+            <td>Header value (Literal).</td>
+            <td>mandatory within <code>mqv:MessageOption</code>)</td>
+            <td>
+             One of <code>0</code>, <code>1</code> or <code>2</code> (only for <code>qos</code>)
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </section>
```
Comment (JKRhb) [inline bindings/protocols/mqtt/index.template.html:108]:
```suggestion
```

File: bindings/protocols/mqtt/index.template.html [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+      <section id="possible-mappings">
+        <h3>Possible mappings</h3>
+      For the MQTT protocol, if an MQTT client publishes a message to a topic with the retain flag set to true, the future subscribers of the topic will also get this message.
+      Outside of this case, it is not possible to read a property but only possible to observe it. 
+      <span class="rfc2119-assertion" id="iana-security-alter">Additionally, in a Form element with MQTT protocol, if the <code>op</code> contains <code>readproperty</code> (meaning that retain flag is set to true), it <em class="rfc2119" title="SHOULD">SHOULD</em> also contain <code>observeproperty</code>.</span>
+      <span class="rfc2119-assertion" id="iana-security-alter">On the other hand, if the MQTT publisher does not set
+        the retain flag to true, the property will be only observable. In this case, the property in the exposed Thing
+        Description <em class="rfc2119" title="SHOULD NOT">SHOULD NOT</em> have Form elements with MQTT protocol containing <code>readproperty</code> operation.</span>
+      </section>
+      </section>
```
Comment (JKRhb) [inline bindings/protocols/mqtt/index.template.html:181]:
```suggestion
```

**Review by JKRhb — COMMENTED** [review]
The PR looks good to me :) I just noticed that there might some duplication of `</section>` tags, see ~~below~~ above.

**Comment by egekorkan** [comment]
Call of 15.12: We will merge it asynchronously after the section tags are fixed

**Comment by relu91** [comment]
@egekorkan should be ready to merge :)

**Comment by egekorkan** [comment]
Thanks @relu91 !

**Pull request merged**